Datascience_With_Python/DS Resources/Deep Learning/Introduction to Pytorch/README.md

```diff
@@ -1,3 +1,39 @@
+
+# Introduction to Pytorch 
+## What is PyTorch?
+#### It’s a Python based package for serving as a replacement of Numpy and to provide flexibility as a Deep Learning Development Platform.
+## Why PyTorch?
+*I encourage you to read Fast AI’s blog post for the reason of the course’s switch to PyTorch.
+Or simply put:*
+* Dynamic Graphs
+* More intuitive than TF (Personal View)
+
```

Comment:
Please add a few applications of the PyTorch framework as well.

Reply:
Add the video link in the README file.

Reply:
Also @prathimacode-hub the content has to be only about PyTorch right? So why is Tensorflow included?